prowler/lib/outputs/json.py

```diff
@@ -55,7 +55,12 @@ def fill_json_asff(finding_output, audit_info, finding, output_options):
             Label=finding.check_metadata.Severity.upper()
         )
         finding_output.Title = finding.check_metadata.CheckTitle
-        finding_output.Description = finding.status_extended
+        # Description should NOT be longer than 1024 characters
+        finding_output.Description = (
+            (finding.status_extended[:1000] + "..")
+            if len(finding.status_extended) > 100
+            else finding.status_extended
+        )
```

Comment:
Could you include this as a part of the class validator? It's so easy to do it with Pydantic https://docs.pydantic.dev/latest/usage/validators/

Reply:
Also, you are doing the comparison with `100` as length.

Reply:
Why do I need validators? I want to change the field, not to validate it.

Reply:
Because you are doing a validation checking the field length, also with that you have all the logic to build the model within the class since you can do that:
```
class JSONASFF(BaseModel):
    Description: str

    @validator('Description')
    def description_sould_not_exceed_1024_chars(cls, value):
        if len(value) > 1000:
            return value[:1000] + ".."
```

Reply:
Also, please include tests that validates the new logic included